src/material.cpp

```diff
@@ -111,6 +102,30 @@ void pyne::Material::_load_comp_protocol1(hid_t db, std::string datapath, int ro
   H5Aread(nuc_attr, str_attr, nucpathbuf);
   nucpath = std::string(nucpathbuf, nuc_attr_len);
   delete[] nucpathbuf;
+  H5Tclose(str_attr);
+  std::cout << "nucpath found in: " << nucpath << std::endl;
```

Comment:
Are these left over from debugging, or do you think the user wants to know this?